dbms/src/Flash/Planner/Plans/PhysicalTableScan.cpp

```diff
@@ -70,30 +72,58 @@ void PhysicalTableScan::buildBlockInputStreamImpl(DAGPipeline & pipeline, Contex
     }
 }
 
+void PhysicalTableScan::buildPipeline(
+    PipelineBuilder & builder,
+    Context & context,
+    PipelineExecutorStatus & exec_status)
+{
+    storage_interpreter = std::make_unique<DAGStorageInterpreter>(
+        context,
+        tidb_table_scan,
+        filter_conditions,
+        context.getMaxStreams());
+    source_ops = storage_interpreter->execute(exec_status);
```

Comment:
Maybe we can hold a tmp `PipelineExecGroupBuilder` in `PhysicalTableScan` and don't need to spilt `storage_interpreter->execute` and `storage_interpreter->executeSuffix`.

Reply:
> Maybe we can hold a tmp `PipelineExecGroupBuilder` in `PhysicalTableScan` and don't need to spilt `storage_interpreter->execute` and `storage_interpreter->executeSuffix`.

But `PipelineExecGroupBuilder` is inited in buildExecGroup, I didn't intend to init `PipelineExecGroupBuilder` in `buildPipeline` which is weird.
And `storage_interpreter->execute` just build `sourceOps`, and executeSuffix build other `transformOps`. I think this separation is clear.

Reply:
ok